python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -623,6 +630,39 @@ def calculateMetrics(self, difference):
                 self.metadata.add("%s_mask_fraction"%maskPlane.lower(), -1)
                 self.log.info("Unable to calculate metrics for mask plane %s: not in image"%maskPlane)
 
+        maglim_science = self._calculateMagLim(science)
+        fluxlim_science = (maglim_science*u.ABmag).to_value(u.nJy)
+        maglim_template = self._calculateMagLim(matchedTemplate)
+        fluxlim_template = (maglim_template*u.ABmag).to_value(u.nJy)
+        maglim_diffim = (np.sqrt(fluxlim_science**2 + fluxlim_template**2)*u.nJy).to(u.ABmag).value
```

Comment:
We might as well add the science and template limiting magnitudes to the metadata, since we have those values handy.